lib/mix/lib/mix/release.ex

```diff
@@ -786,9 +793,9 @@ defmodule Mix.Release do
   The exact chunks that are kept are not documented and may change in
   future versions.
   """
-  @spec strip_beam(binary()) :: {:ok, binary} | {:error, :beam_lib, term}
-  def strip_beam(binary) do
-    case :beam_lib.chunks(binary, @significant_chunks, [:allow_missing_chunks]) do
+  @spec strip_beam(binary(), List.t()) :: {:ok, binary} | {:error, :beam_lib, term}
+  def strip_beam(binary, exceptions) do
+    case :beam_lib.chunks(binary, @significant_chunks ++ exceptions, [:allow_missing_chunks]) do
```

Comment:
We probably want to do:

```suggestion
    case :beam_lib.chunks(binary, Enum.uniq(@significant_chunks ++ exceptions), [:allow_missing_chunks]) do
```